src/main/java/annotator/model/pack/PackageImporter.java

```diff
@@ -0,0 +1,100 @@
+package annotator.model.pack;
+
+import annotator.model.type.Type;
+import annotator.model.type.TypeCreator;
+import annotator.model.word.WordCreator;
+import au.com.bytecode.opencsv.CSVReader;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.*;
+
+public class PackageImporter {
+
+    private final PackageCreator packageCreator;
+    private final WordCreator wordCreator;
+    private final TypeCreator typeCreator;
+
+    public PackageImporter(
+        PackageCreator packageCreator,
+        WordCreator wordCreator,
+        TypeCreator typeCreator
+    ) {
+        this.packageCreator = packageCreator;
+        this.wordCreator = wordCreator;
+        this.typeCreator = typeCreator;
+    }
+
+    public List<Package> importPackage(String typeName, String packageName, InputStream file) throws InvalidFileFormatException, IOException, PackageCreateConflictException {
+
+        Map<String, Boolean> words = this.parseWords(file);
```

Comment:
are we sure that no word can be inserted two times? I would use list of pairs<String, Boolean>. If we are certainly sure that there are no duplicates required, then this is fine.